packages/wallet-sdk/src/createCoinbaseWalletSDK.ts

```diff
@@ -67,5 +67,13 @@ export function createCoinbaseWalletSDK(params: CreateCoinbaseWalletSDKOptions)
       }
       return provider;
     },
+    accounts: {
+      setSigner(params: SubAccountState['getSigner']) {
```

Comment:
is this the setSigner changes you made for me to try?

Reply:
Yes. I debated leaving it in